smartrack-manifest.xml

```diff
@@ -9,26 +9,41 @@
   <project name="meta-freescale-3rdparty.git" path="layers/meta-freescale-3rdparty" remote="githf" revision="05b1746a4adc240b690fe965ac5b8a043d9b9d03" upstream="dunfell"/>
+  <!-- meta-freescale-distro: Branch dunfell -->
   <project name="meta-freescale-distro.git" path="layers/meta-freescale-distro" remote="githf" revision="5d882cdf079b3bde0bd9869ce3ca3db411acbf3b" upstream="dunfell"/>
+  <!-- meta-freescale: Branch dunfell -->
   <project name="meta-freescale.git" path="layers/meta-freescale" remote="githf" revision="46f54fdc5ad854a22bf759aac7fd65db1d1bb574" upstream="dunfell"/>
   
-  <project name="meta-toradex-bsp-common.git" path="layers/meta-toradex-bsp-common" remote="tdx" revision="cab1ac409fa532c22f64c570f1b5c2350b570f94" upstream="dunfell-5.x.y"/>
-  <project name="meta-toradex-nxp.git" path="layers/meta-toradex-nxp" remote="tdx" revision="4087b01cd5c70f4656cdff26a4e7d7ecc73bf41f" upstream="dunfell-5.x.y"/>
-  <project name="meta-toradex-tegra.git" path="layers/meta-toradex-tegra" remote="tdx" revision="b5179b050062a29c2fe0bdfc1717b616c4f1630e" upstream="dunfell-5.x.y"/>
-
+  <!-- meta-toradex-bsp-common: Branch dunfell-5.x.y -->
+  <project name="meta-toradex-bsp-common.git" path="layers/meta-toradex-bsp-common" remote="tdx" revision="cdbfe017be9096adbbfb4d3bbea0421e44f54cf3" upstream="dunfell-5.x.y"/>
+  <!-- meta-toradex-nxp: Branch dunfell-5.x.y -->
+  <project name="meta-toradex-nxp.git" path="layers/meta-toradex-nxp" remote="tdx" revision="8e762e3c8ec5c84f76254f442f37c70dfb7e1b68" upstream="dunfell-5.x.y"/>
+  <!-- meta-toradex-tegra: Branch dunfell-5.x.y -->
+  <project name="meta-toradex-tegra.git" path="layers/meta-toradex-tegra" remote="tdx" revision="b214067b27204f471cf405a1e3997d3da233a8f9" upstream="dunfell-5.x.y"/>
+  
+  <!-- meta-qt5: Branch dunfell -->
   <project name="meta-qt5.git" path="layers/meta-qt5" remote="githq" revision="0d8eb956015acdea7e77cd6672d08dce18061510" upstream="dunfell"/>
-  <project name="meta-toradex-demos.git" path="layers/meta-toradex-demos" remote="tdx" revision="b962519c754bd918e99ff08e583f63ffb10757b0" upstream="dunfell-5.x.y"/>
+  <!-- meta-toradex-demos: Branch dunfell-5.x.y -->
+  <project name="meta-toradex-demos.git" path="layers/meta-toradex-demos" remote="tdx" revision="38705bdc1ff671597680c5a9c9d1a76822bb5620" upstream="dunfell-5.x.y"/>
+  <!-- meta-toradex-distro: Branch dunfell-5.x.y -->
   <project name="meta-toradex-distro.git" path="layers/meta-toradex-distro" remote="tdx" revision="48913ab629a231913e3e147e93e57f7cf6019ac6" upstream="dunfell-5.x.y"/>
-    
+  
+  <!-- meta-smartrack: Branch main -->
   <project name="meta-smartrack.git" path="layers/meta-smartrack" remote="nismrck" revision="main" upstream="main">
     <copyfile dest="export" src="buildconf/export"/>
   </project>
   
+  <!-- meta-smartrack: Branch dunfell -->
   <project name="meta-rauc.git" path="layers/meta-rauc" remote="rauc" revision="dunfell" upstream="dunfell"/>
```

Comment:
Question: The revision is refer to "denfell" instead of a commit hash, is that means it will always pull the latest commit from the dunfell branch? 

Some context:
We need to make sure each build is reproducible (for reference and debugging purpose), meaning if someone tell us "hey your RCU Image version 1.0.0d23 fail to boot", then we need to reproduce this build and debug the problem. 
Hence, we need a way to know "for build number 99, I'm checking out branch X commit Y from repo A; branch P commit Q from repo B, etc".  I saw RTOS team generate [txt files](https://p4.natinst.com/browser/perforce/build/exports/ni/nioe/NIOE_Dunfell_x64_temp/official/export/20.7/20.7.0d112/targets/linuxU/x64/gcc-4.7-oe/release/) to store all these info as part of the building process, perhaps we can follow their way or you can think of something creative (e.g. create a script for the AzDO pipeline to execute, etc).

Reply:
@karhin1990 Yes, it was pointing to the latest commit in dunfell branch. I'm changing it to point to a specific commit for now . I'll start to think about how we handle and track these things.

Reply:
I will mark this as resolved. For the reproducible build matter, please create a task in either of the user story you are working on currently to track it. Thanks!